bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/package-info.java

```diff
@@ -1,5 +1,5 @@
 /*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
- ~ Copyright 2017 Adobe Systems Incorporated
+ ~ Copyright 2019 Adobe Systems Incorporated
```

Comment:
It should still be 2017, the year this file was created.

Reply:
review comments incorporated